salesagility/SuiteCRM
# Mass update on user preferences doesn't save 7.8

Moved code from save.php to user.php and deleted save.php removed calls
to deleted file in controller.php



## Description

Added function saveFormPreferences() to User.php whcih contains code from deleted file save.php and removed call to save.php in controller.


Issue: #6402
## Motivation and Context

Mass update on user preference fields doesn't save
## How To Test This

-Navigate to admin panel
-Select users to mass update
-Change Email Client field
-Check if change has saved in user edit view

## Types of changes


### Final checklist
